### ISSUE TITLE
trying to get slot "defined" from an object of a basic class ("environment") with no slots

### ISSUE THREAD

**Issue by sneumann** [sneumann/xcms#531]
Hi,
we have an ERROR in http://bioconductor.org/checkResults/devel/bioc-LATEST/xcms/malbec2-checksrc.html
```
...
* checking whether the namespace can be unloaded cleanly ... WARNING
---- unloading
Error in .mergeMethodsTable(generic, mtable, tt, attach) : 
  trying to get slot "defined" from an object of a basic class ("environment") with no slots
Calls: unloadNamespace ... <Anonymous> -> .updateMethodsInTable -> .mergeMethodsTable
Execution halted
```

**Comment by sneumann** [comment]
Hm, can't confirm for 3.13.3 inside ``
with 
```
R version 4.0.3 (2020-10-10)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 20.04 LTS

Matrix products: default
BLAS/LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/libopenblasp-r0.3.8.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=C             
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats4    parallel  stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] xcms_3.13.3         MSnbase_2.16.0      ProtGenerics_1.22.0
 [4] S4Vectors_0.28.1    mzR_2.24.1          Rcpp_1.0.5         
 [7] BiocParallel_1.24.1 Biobase_2.50.0      BiocGenerics_0.36.0
[10] BiocManager_1.30.10

loaded via a namespace (and not attached):
 [1] SummarizedExperiment_1.20.0 lattice_0.20-41            
 [3] colorspace_2.0-0            vctrs_0.3.4                
 [5] vsn_3.58.0                  XML_3.99-0.5               
 [7] rlang_0.4.8                 pillar_1.4.6               
 [9] glue_1.4.2                  affy_1.68.0                
[11] RColorBrewer_1.1-2          affyio_1.60.0              
[13] matrixStats_0.57.0          GenomeInfoDbData_1.2.4     
[15] foreach_1.5.1               lifecycle_0.2.0            
[17] plyr_1.8.6                  mzID_1.28.0                
[19] robustbase_0.93-6           zlibbioc_1.36.0            
[21] MatrixGenerics_1.2.0        munsell_0.5.0              
[23] pcaMethods_1.82.0           gtable_0.3.0               
[25] codetools_0.2-16            IRanges_2.24.1             
[27] doParallel_1.0.16           GenomeInfoDb_1.26.2        
[29] MassSpecWavelet_1.56.0      preprocessCore_1.52.0      
[31] DEoptimR_1.0-8              scales_1.1.1               
[33] limma_3.46.0                DelayedArray_0.16.0        
[35] XVector_0.30.0              MsCoreUtils_1.2.0          
[37] RANN_2.6.1                  impute_1.64.0              
[39] ggplot2_3.3.2               digest_0.6.27              
[41] ncdf4_1.17                  GenomicRanges_1.42.0       
[43] grid_4.0.3                  tools_4.0.3                
[45] bitops_1.0-6                magrittr_1.5               
[47] RCurl_1.98-1.2              tibble_3.0.4               
[49] crayon_1.3.4                pkgconfig_2.0.3            
[51] MASS_7.3-53                 ellipsis_0.3.1             
[53] Matrix_1.2-18               iterators_1.0.13           
[55] R6_2.5.0                    MALDIquant_1.19.3          
[57] compiler_4.0.3    
```

**Comment by sneumann** [comment]
Ok, I can confirm on Rdevel:
```
R Under development (unstable) (2020-12-24 r79679)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 20.04.1 LTS

Matrix products: default
BLAS/LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/libopenblasp-r0.3.8.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=C             
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats4    parallel  stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] xcms_3.13.0         MSnbase_2.17.3      ProtGenerics_1.23.5
 [4] S4Vectors_0.29.6    mzR_2.25.1          Rcpp_1.0.5         
 [7] BiocParallel_1.25.2 Biobase_2.51.0      BiocGenerics_0.37.0
[10] BiocManager_1.30.10

loaded via a namespace (and not attached):
 [1] SummarizedExperiment_1.21.1 lattice_0.20-41            
 [3] colorspace_2.0-0            vctrs_0.3.6                
 [5] vsn_3.59.0                  XML_3.99-0.5               
 [7] rlang_0.4.9                 pillar_1.4.7               
 [9] glue_1.4.2                  affy_1.69.0                
[11] RColorBrewer_1.1-2          affyio_1.61.0              
[13] matrixStats_0.57.0          GenomeInfoDbData_1.2.4     
[15] foreach_1.5.1               lifecycle_0.2.0            
[17] plyr_1.8.6                  mzID_1.29.0                
[19] robustbase_0.93-6           zlibbioc_1.37.0            
[21] MatrixGenerics_1.3.0        munsell_0.5.0              
[23] pcaMethods_1.83.0           gtable_0.3.0               
[25] codetools_0.2-18            IRanges_2.25.6             
[27] doParallel_1.0.16           GenomeInfoDb_1.27.3        
[29] MassSpecWavelet_1.57.0      preprocessCore_1.53.1      
[31] DEoptimR_1.0-8              scales_1.1.1               
[33] limma_3.47.3                DelayedArray_0.17.6        
[35] XVector_0.31.1              MsCoreUtils_1.3.2          
[37] RANN_2.6.1                  impute_1.65.0              
[39] ggplot2_3.3.2               digest_0.6.27              
[41] ncdf4_1.17                  GenomicRanges_1.43.1       
[43] grid_4.1.0                  tools_4.1.0                
[45] bitops_1.0-6                magrittr_2.0.1             
[47] RCurl_1.98-1.2              tibble_3.0.4               
[49] crayon_1.3.4                pkgconfig_2.0.3            
[51] MASS_7.3-53                 ellipsis_0.3.1             
[53] Matrix_1.3-0                iterators_1.0.13           
[55] R6_2.5.0                    MALDIquant_1.19.3          
[57] compiler_4.1.0             
```

**Comment by sneumann** [comment]
Seems to be fixed in 3.13.4 (3b7d0ef50647ed3986d138c5bdfbe32bc2bcbcad), just pushed that to BioC. Yours, Steffen

**Comment by sneumann** [comment]
confirmed, works in `3.13.4`. Thanks Jo for fixing ! Yours, Steffen
